code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc

```diff
@@ -0,0 +1,373 @@
+== Monitor and Fix Issues in Your Scans
+
+toc::[]
+
+Prisma Cloud performs periodic scans on each integrated repository of the Version Control System (VCS) and CI/CD to find infrastructure misconfigurations.
+
+*Projects* provides a consolidated view of scan results across your repositories, CI/CD and packages wherein categorization of issues are at resource or policy level giving you a better understanding on your security posture.
+
+Prisma Cloud performs three types of scans on your integrations.
+
+* *VCS main branch scans*: Periodic scans performed on all main branches across repositories.
+* *VCS Pull Requests*: Periodic scans using Enforcement parameters are run on all open Pull Requests (PR) for your integrated repositories.
+* *CLI and CI/CD runs*: The Enforcement parameters with periodic scans are performed on all runs for your integrated repositories.
+
+All three types of scans are run on code categories of IaC misconfiguration, Vulnerabilities, Secrets, Licenses and Build Integrity.
+
+image::proj.png[width=800]
+
+[cols="1,2", options="header"]
+|===
+
+| Projects Code Category View
+| Description
+
+|Overview
+|A summary view of all your scan results across all code categories. The Overview lists errors by prioritizing build issues across the integrations.
+
+|IaC Misconfiguration
+|The view lists security issues after scanning integrated repositories, wherein the master branch of a repository lists all code security violations in the resource block with code tags, resource dependencies and resource history. In this view, you can also identify Drift and trigger Manual Scans for your repository.
```

Comment:
let's call it "default" branch and not "master"

Reply:
trigger manual scans is across all views - this is an action from the upper menu